McAfee/mcafee_web_gateway/tests/mcafee_access_log_blocked.json

```diff
@@ -0,0 +1,62 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "mcafee-web-gateway",
+        "dialect_uuid": "40bac399-2d8e-40e3-af3b-f73a622c9687"
+      }
+    },
+    "message": "date=\"2022-03-11T10:39:16.390Z\" hostname=\"mwgproxy\" username=\"\" source_ip=1.2.3.4 destination_ip=2.2.2.41 destination_host=\"www.forbiddensite.com\" http_status_code=403 media_type=\"\" source_bytes=131 destination_bytes=0 http_request_first_line=\"GET http://www.forbiddensite.com/ HTTP/1.1\" url_categories=\"Pornography\" url_reputation_string=\"Minimal Risk\" url_reputation_code=-28 ruleset_name=\"Default\" rule_name=\"Block URLs Whose Category Is in Category Blocklist for Default Groups\" block_id=10 block_reason=\"Blocked by URL filtering\" body_infected=false virus_names=\"\" body_modified=false application_reputation=\"Unverified\" application_name=\"forbiddenapp\" http_referer=\"\" user_agent=\"curl/7.77.0\""
+  },
+  "expected": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "mcafee-web-gateway",
+        "dialect_uuid": "40bac399-2d8e-40e3-af3b-f73a622c9687",
+        "parsing_status": "success"
+      }
+    },
+    "message": "date=\"2022-03-11T10:39:16.390Z\" hostname=\"mwgproxy\" username=\"\" source_ip=1.2.3.4 destination_ip=2.2.2.41 destination_host=\"www.forbiddensite.com\" http_status_code=403 media_type=\"\" source_bytes=131 destination_bytes=0 http_request_first_line=\"GET http://www.forbiddensite.com/ HTTP/1.1\" url_categories=\"Pornography\" url_reputation_string=\"Minimal Risk\" url_reputation_code=-28 ruleset_name=\"Default\" rule_name=\"Block URLs Whose Category Is in Category Blocklist for Default Groups\" block_id=10 block_reason=\"Blocked by URL filtering\" body_infected=false virus_names=\"\" body_modified=false application_reputation=\"Unverified\" application_name=\"forbiddenapp\" http_referer=\"\" user_agent=\"curl/7.77.0\"",
+    "source": {"ip": "1.2.3.4"},
+    "destination": {"ip": "2.2.2.41"},
+    "event": {
+        "action": "block",
+        "start": "2022-03-11T10:39:16.390Z",
+        "code": "10",
+        "reason": "Blocked by URL filtering"
```

Comment:
You are missing an ECS categorization. For blocked URLs, you could use `event.type="denied"` and `event.category="network"`.

More information on ECS categorization fields: https://www.elastic.co/guide/en/ecs/current/ecs-category-field-values-reference.html

Reply:
Fixed